### ISSUE TITLE
groups: error on reload "Failed to fetch dynamically imported module: https://fabled-faster.tlon.network/apps/groups/assets/DiaryAddNote.7d3a597c.js"

### ISSUE THREAD

**Issue by urcades** [tloncorp/landscape-apps#1001]
Received the following error output while attempting to reload the interface

```
TypeError: Failed to fetch dynamically imported module: https://fabled-faster.tlon.network/apps/groups/assets/DiaryAddNote.7d3a597c.js
```

**Additional context:**
- Ship: planet (~fabled-faster)
- Instance type: Tlon-hosted
- MacOS Monterey — 12.6 (21G115)
- Chromium — Engine Version 106.0.5249.103

**Comment by arthyn** [comment]
Finally figured this one out, this only happens if the app updates behind the scenes while you have a tab open. We should be able to guard against this but I think it'll require a refresh from the user.

**Comment by lukestiles** [comment]
Will we be able to identify this state so we can message the user they need to refresh?

**Comment by arthyn** [comment]
There's a few ways we can do this, but it'll take a little bit of work.

**Comment by arthyn** [comment]
Closed in favor of more recent one, #1733
